### ISSUE TITLE
failed to install mongodb on WSL 

### ISSUE THREAD

**Issue by MaxwellWibert** [MicrosoftDocs/windows-uwp#2478]
When following instructions in this tutorial for mongodb
"sudo apt-get mongodb" gives error "Package mongodb has no installation candidate"
WSL running with pengwin distro
Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb282339-e2e5-ae64-4d76-3c999828f4e4
* Version Independent ID: f7c88e68-43a2-bc01-73c3-a64b2aa198fd
* Content: [Get started using Python on Windows with a database](https://docs.microsoft.com/en-us/windows/python/databases)
* Content Source: [hub/python/databases.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/python/databases.md)
* Product: **dev-environment**
* Technology: **windows-python**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

**Comment by mattwojo** [comment]
Hmm.. thanks @MaxwellWibert. I don't currently have Pengwin but just requested a copy to test. I found this issue in the Pengwin repo: https://github.com/WhitewaterFoundry/Pengwin/issues/469
They suggest trying an update first: 
sudo apt update && sudo apt install mongodb 

Could you give this a try and let me know if it resolves the problem? 
Thanks again,
Matt

**Comment by MaxwellWibert** [comment]
@mattwojo I'm afraid that didn't work. same error message after the update finishes

**Comment by mattwojo** [comment]
Bummer. I got a copy of Pengwin installed and took a look. I was able to install MongoDB in Pengwin following the instructions for Debian (which Pengwin is based on) from the Mongo site: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-debian/

Basically ran this: `echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/4.2 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list`

Then ran `sudo apt-get update` and `sudo apt-get install -y mongodb-org`

This seems to have successfully installed. I still had some troubles starting the service though. 

The install seems to work fine on WSL in other distros, so I'm closing this issue and recommend filing eiher filing a new issue with the Pengwin repo or adding on to the exitsting issue that I linked above as it looks like an update on their end: https://github.com/WhitewaterFoundry/Pengwin

Thanks for bringing this to our attention. I let the WSL PM know as well.
